packages/parser/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export type { Info } from './language/index.js';
+export type { Info, Packet } from './language/index.js';
```

Comment:
I'd prefer to export the main AST types from parser and extract the necessary types downstream. 
I've added `RecursiveAstOmit` and have used that.

Reply:
I think in some point we would need to export more attributes from the parser packages. I'm currently working on language server in a separate repository, and all the attributes below are necessary there:

<https://github.com/mermaid-js/mermaid/blob/e86f09aaedce59af5c4edc890fcd1a11dc04432f/packages/parser/src/language/index.ts#L1-L15>

And that's what I've exported in #4751, but in term of parser it would fine to not export them.